mingw-w64-nss/PKGBUILD

```
@@ -10,6 +10,7 @@ pkgdesc="Mozilla Network Security Services (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
 url="https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS"
+install=nss-${CARCH}.install
```

```suggestion
install=nss-${MSYSTEM}.install
```
Please, rename/add the required files and edit accordingly, for example install-UCRT64 with _mingw=ucrt64

---

@MehdiChinoune So when we do a .install script now we have to make six copies?

---

Yes.

---

Please rename install-i686 and add install-CLANG32, even you couldn't test them.

---

where is nss-MINGW32.install?